auto_gptq/modeling/_base.py

```diff
@@ -370,28 +368,10 @@ def tmp(_, inp, out):
                             static_groups=self.quantize_config.static_groups,
                         )
 
-                        stat = {"layer": i + 1, "module": name, "avg_loss": avg_loss, "time": duration}
+                        stat = {"layer": i, "module": name, "avg_loss": avg_loss, "time": duration}
 
-                        # generate metric diff
```

Comment:
@fxmarty I believe the grid printing (tabulate) and coloring of diff (termcolor) dependencies caused this part of the to be removed. Is there a way to work around this? This is very important to save time in quant validation/testing. gptq quanting is very sensitive to params (gpsize, dam, nsamples) etc and extremely long. It can take several hours to complete a quant. Now, users/tests can easier eye-ball the avg_loss as the logs stream and do a in-brain diff, or wait for quant to complete to do PPL. Both are doable but inefficient. Having a runtime diff of a saved baseline (quant-log) anyone can tell the new quant, due to whatever param change, is going to be bad in only few early layers and early stop quant.

Reply:
We could just add the ability to pass previous logs, and print the diff with it. I would just like to avoid to add dependencies.

Is this fine to do in an other PR? This one is already huge.